src/main/scala/org/constellation/infrastructure/configuration/CliConfigParser.scala

```diff
@@ -44,17 +44,35 @@ object CliConfigParser {
       opt[Unit]('t', "test-mode")
         .action((x, c) => c.copy(testMode = true))
         .text("Run with test settings"),
+      opt[String]('k', "keystore")
```

Comment:
As mentioned above you don't need it if you will use typesafe config

Reply:
Will do in another PR regarding terraform/gcp bcause I need to secure it anyway